### ISSUE TITLE
Bye Bye --- hello ;

### ISSUE THREAD

**Issue by machaval** [mulesoft-labs/data-weave-rfc#16]
One thing that has been going over my mind is to remove `---` or at least make it optional. Every time I have try it I was not able to do it because of grammar ambiguity. But one option that we never explore is to add `;` at the end of the sentences. Then `---` can be omitted. 

To be honest I don't have any problem with `---` but new people find this quite weird. What do we think?

**Comment by thisisverytricky** [comment]
@machaval I agree - the `---` isn't an issue. It is a bit weird at first glance because I don't know any other language with something like it, but it isn't a difficult concept to grasp. I don't see it as a problem, personally.

**Comment by jerneyio** [comment]
I didn't like it at first, but it's grown on me. I currently like that it makes it more obvious whether the file I'm dealing with is a module or an executable script

**Comment by menduz** [comment]
what if you replace it with a top level return instead?

```dwl
%dw 2.0

fun myThing() = 1

return {a: myThing()}
```
